.github/workflows/hello-world.yaml

```diff
@@ -0,0 +1,19 @@
+name: "hello world workflow"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+
+permissions:
+  actions: read
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  hello-world:
+    runs-on: ["in-service", "n150"]
```

Comment:
what does "in-service" means?

Reply:
the label in-service, just means that the runner is in service.
Often times we need to do maintainence so we use this label to enable/disable the runner from executing CI jobs